tests/models/rag/test_modeling_rag.py

```diff
@@ -653,7 +653,7 @@ class RagDPRT5Test(RagTestMixin, unittest.TestCase):
     def config_and_inputs(self):
         question_encoder_tester = DPRModelTester(self)
         dpr_config_and_inputs = question_encoder_tester.prepare_config_and_inputs()
-        generator_tester = T5ModelTester(self, vocab_size=1100)
+        generator_tester = T5ModelTester(self, vocab_size=1101)
```

Comment:
If we go check the test tokenizer, the last token has index `1100`, so the vocab size has to be `1101` otherwise we will get indexing errors (causing the red CI)